server/protocols/jmap/src/test/java/org/apache/james/jmap/model/MessageFactoryTest.java

```diff
@@ -213,7 +219,13 @@ public void headersShouldBeUnfoldedAndDecoded() throws Exception {
             .preview("(Empty)")
             .textBody(Optional.of(""))
             .htmlBody(Optional.empty())
+            .isAnswered(false)
```

Comment:
No, this case is just expected that we can convert enough data from MetaDataWithContent to JmapMessage so yes, it's duplicated because we are supporting both isFlag and Keywords